packages/palette/src/elements/Cards/CardTag.tsx

```diff
@@ -1,9 +1,33 @@
 import React from "react"
+import { color } from "../../helpers"
+import { BorderBox } from "../BorderBox"
+import { Sans } from "../Typography"
 import { CardTagProps } from "./CardTag.shared"
 
 /**
  * `CardTag` is used for the Cards, and is controlled by their `tag` prop.
  */
-export const CardTag: React.FC<CardTagProps> = () => {
-  return null
+export const CardTag: React.FC<CardTagProps> = ({
+  text,
+  textColor,
+  color: bgColor,
+  borderColor,
+}) => {
+  return (
+    <BorderBox
+      borderRadius="2"
+      overflow="hidden"
+      backgroundColor={color(bgColor) || bgColor}
+      borderColor={color(borderColor) || borderColor}
+      m={0}
+      p={0}
+      position="absolute"
```

Comment:
I think for now we only want to display this absolutely positioned over the card. So was not sure why we needed the complications with passing `style` there. For now hardcoding this position here.

Reply:
So the reason for this is because the Small card needs the left and top margin to be 10px, the Medium one needs 15px, and the Large needs 20px. In the ios versions of the cards you can see how `space(x)` differs on `<CardTag ..`, so we should support this additional styling here.

Reply:
I've updated the display to give a background. Not sure if that is better since it's a bit confusing if it's intentional or a bug, but definitely gives a better idea of boundary and changes when resizing.

Not sure why I don't see an update with storybook here, but the look:
<img width="824" alt="Screen Shot 2020-07-07 at 12 20 37 PM" src="https://user-images.githubusercontent.com/437156/86833945-10801400-c04f-11ea-83b5-5fb91b8fcbe2.png">